README.md

```diff
@@ -667,18 +809,40 @@ function vendingMachine (snackSelection, moneyInserted) {
 }
 ```
 
-If the selection isn't valid, `return` immediately with a failure message. Otherwise, invoke the `getPrice()` function and store the returned number in the `price` variable. Then, if the `price` is higher than the amount of `moneyInserted`, `return` with a failure message. Otherwise, calculate the change, and then `return` a success message.
-
-We could continue to refactor this into smaller pieces, but this is already a great start. And, frankly, for such simple code, there's not much need to break it down further. It's already in much more manageable pieces than we started out with, and it's absolutely possible to write code that's **too** small and modular. For example, we could have extracted the calculation of `change` into its own function:
+If the selection isn't valid, `return` immediately with a failure message.
+Otherwise, invoke the `getPrice()` function and store the returned number in the
+`price` variable. Then, if the `price` is higher than the amount of
+`moneyInserted`, `return` with a failure message. Otherwise, calculate the
+change, and then `return` a success message.
+
+We could continue to refactor this into smaller pieces, but this is already a
+great start. And, frankly, for such simple code, there's not much need to break
+it down further. It's already in much more manageable pieces than we started out
+with, and it's absolutely possible to write code that's **too** small and
+modular. For example, we could have extracted the calculation of `change` into
```

Comment:
> We could continue to refactor this into smaller pieces, but this is already a great start. There's not much need to break it down further since already in much more manageable pieces than we started out with.  It's absolutely possible to write code that's *too* small and modular.